### ISSUE TITLE
Cleaning: replacing `format` by formatted strings

### ISSUE THREAD

**Issue by Arthur-Milchior** [ankidroid/Anki-Android#11449]
This is part of Kotlin clean-up  #10489, low priority, a way to slightly improve code quality and make easy changes.

There are currently 37 occurrences of `String.format` in Kotlin followed by a String literal. In Kotlin, you can generally use Template stings, which I find more readable. It'd be nice to replace them. In short, if you don't know string literals, you just have to use $varName or ${expression}. So for example you'd use `"$appName $pkgVersionName"` instead of `"String.format("%s v%s", appName, pkgVersionName)"`.

If `String.format` don't start with a template but with a locale, please don't change it, unless you can be sure that:
* either we don't actually care about the locale
* you can ensure we respect the locale requirement directly in the parameters

If it's not too hard, I appreciate if you can do one change by pull request; unless multiple changes are very strongly related. This way, those PR can be merged very quickly; and if there is an issue, I don't have to delay all other changes.

You don't have to ask permission to work on this. Just look at the answer below to see links to PR already done. I'll try to merge them quickly. Hopefully, most change should be done in less than a minute, so the risk of wasted work is really small

**Comment by sparkyniner** [comment]
Can I work on this?

**Comment by mikehardy** [comment]
@sparkyniner please note this in the description:

---

If it's not too hard, I appreciate if you can do one change by pull request; unless multiple changes are very strongly related. This way, those PR can be merged very quickly; and if there is an issue, I don't have to delay all other changes.

**You don't have to ask permission to work on this**. Just look at the answer below to see links to PR already done. I'll try to merge them quickly. Hopefully, most change should be done in less than a minute, so the risk of wasted work is really small

---

emphasis added

**Comment by sparkyniner** [comment]
Was busy on another PR, will work on this asap.

**Comment by sparkyniner** [comment]
Hello, I have issued a PR and changed one instance in one file as requested, if this ok I can change the other 36 instances.

**Comment by Arthur-Milchior** [comment]
I just edited the issue message to be explicit about locales. Essentially that this issue is mostly about `format` that are applied directly to a format string, and is not about ones where locale are explicit.

**Comment by BrayanDSO** [comment]
IMO, this should be closed because:
1. It is open-ended
2. The added value is infimal
3. It becomes harder with each removal of String.format
4. Our human resources (contributors and reviewers' time) are quite limited, so it's better to spend it on other issues

I'll give at least a week for anyone else to disagree. If there are no complaints, I'll be closing this

**Comment by itsAfnanAlam** [comment]
is somebody working on this?

**Comment by BrayanDSO** [comment]
> is somebody working on this?

This issue is going to be closed, so there shouldn't be anyone working on this
